### DISCUSSION TITLE
Change default behaviour AUTH_FAILED

### DISCUSSION THREAD

**Discussion by tenner18** [schwabe/ics-openvpn#1401] [Q&A]
To make issues more manageable, I would appreciate it if you fill out the following details as applicable:

# General information
1. Android Version 
Android 11
2. Android Vendor/Custom ROM
4.19.157-perf+ (Motorola)
3. Device
Moto G100
4. Version of the app (version number/play store version/self-built)
0.7.27
 
# Description of the issue
Is there a way to change the default behaviour of AUTH_FAILED when importing a profile, maybe by a parameter inside the profile? I want to set it to "Disconnect, save password"

**Comment by schwabe** [comment]
See `auth-retry`
